Osteppy/package.json

```diff
@@ -4,16 +4,31 @@
   "description": "Slack micro-service",
   "main": "dist",
   "scripts": {
-    "dev": "nodemon -w src --exec \"babel-node src/server.js --presets es2015,stage-0\"",
-    "build": "babel src -s -D -d dist --presets es2015,stage-0",
-    "start": "node dist",
+    "dev": "nodemon -w src --exec \"babel-node src/server.js\"",
+    "build": "babel src --copy-files --source-maps=true --out-dir dist",
+    "start": "node dist/server.js",
```

Comment:
I was a little confused here since I get a module load error where node thinks `dist` is a module rather than a directory. I fixed it so it works, but please verify the intention.